babel.config.js

```diff
@@ -1,7 +1,12 @@
 module.exports = {
   presets: [
     'module:metro-react-native-babel-preset',
-    'module:react-native-dotenv'
+    'module:react-native-dotenv',
+    '@babel/preset-env', {
+      targets: {
+        node: 'current'
+      }
+    }
```

Comment:
Be careful, you're compiling for a Browser, not `node`. If you did this to enable support of `async/await` (`regeneratorRuntime undefined` error), you should only add the following to `package.json`. ParcelJS will then compile the bundle with Promises support:
```JSON
"browserslist": [
  "last 1 Chrome versions"
]
```